.github/workflows/release.yml

```diff
@@ -946,7 +946,7 @@ jobs:
           message: Latest full release
       - name: Dispatch Auto-signer Workflow
         if: env.TEST_RUN != 'true'
-        uses: actions/github-script@v7
+        uses: actions/github-script@v8
```

Comment:
@sbendar we need check this also in the allowed actions. In the release notes it says that the minimum compatible runner version is **v2.327.1** Do you know what is the version we are using for runners?

https://github.com/actions/github-script/releases/tag/v8

Reply:
This looks like its covered by us allowed all github owned actions.